mlflow/R/mlflow/.create-test-env.R

```diff
@@ -15,7 +15,11 @@ reticulate::conda_install("xgboost", envname = mlflow:::mlflow_conda_env_name())
 # Pin h2o to prevent version-mismatch between python and R
 reticulate::conda_install("h2o==3.30.1.3", envname = mlflow:::mlflow_conda_env_name(), pip = TRUE)
 
+# Increase timeout because `spark_install` is flaky with the default value (60 seconds)
+options(timeout=60 * 30)
+
 # Install MLeap runtime and required dependencies
-sparklyr::spark_install(version = "2.4.5")
+sparklyr::spark_install(version = "2.4.5", verbose = TRUE)
+# Downloaded spark archive will be cached by GitHub Actions (See the r job in `.github/workflows/master.yml`)
```

Comment:
```
Tue, 02 Mar 2021 14:08:20 GMT
Installing Spark 2.4.5 for Hadoop 2.7 or later.
Tue, 02 Mar 2021 14:08:20 GMT
Downloading from:
Tue, 02 Mar 2021 14:08:20 GMT
- 'https://archive.apache.org/dist/spark/spark-2.4.5/spark-2.4.5-bin-hadoop2.7.tgz'
Tue, 02 Mar 2021 14:08:20 GMT
Installing to:
Tue, 02 Mar 2021 14:08:20 GMT
- '~/spark/spark-2.4.5-bin-hadoop2.7'
Tue, 02 Mar 2021 14:08:20 GMT
trying URL 'https://archive.apache.org/dist/spark/spark-2.4.5/spark-2.4.5-bin-hadoop2.7.tgz'
Tue, 02 Mar 2021 14:08:21 GMT
Content type 'application/x-gzip' length 232530699 bytes (221.8 MB)
Tue, 02 Mar 2021 14:35:56 GMT
==================================================
Tue, 02 Mar 2021 14:35:56 GMT
downloaded 221.8 MB
```

Reply:
30 min is very long but once the archive is cached, we can skip the donwload.

Reply:
https://github.com/mlflow/mlflow/runs/2013929376#step:13:1130

It turns out 30 min. is insufficient.

Reply:
Updated the timeout to one hour which I think is sufficient for `spark_install` to complete.